### PR TITLE
[TGER-59] Migrate Nova resource

### PR DESCRIPTION
**Pull request by phuclh**
- Bump dependencies to the latest version.
- Migrate Nova migration.

### CODE REVIEW

File: src/Nova/Invoice.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            DateTime::make('Created At')->exceptOnForms(),
+            nova('user') ? BelongsTo::make('Updated By', 'updater', nova('user'))->exceptOnForms() : null,
+            DateTime::make('Updated At')->exceptOnForms(),
+        ]);
+    }
+
+    public function filters(Request $request)
+    {
+        return [
+            new Filters\OrderLocation,
```
Comment (phuclh) [inline src/Nova/Invoice.php]: @drewroberts Which package contains this filter? Thank you.

Comment (drewroberts) [reply src/Nova/Invoice.php]: I don't know yet. It will probably end up in the https://github.com/tipoff/locations package

Comment (drewroberts) [reply src/Nova/Invoice.php]: Yeah, let's go with that one

**Review by drewroberts — APPROVED** [review]
Thank you! 🌾

**Pull request merged**